integration-test/1491-mexican-shields.py

```diff
@@ -0,0 +1,1074 @@
+from . import FixtureTest
+
+
+class MexicanShieldText(FixtureTest):
+    def test_mx_mex_prefix(self):
+        import dsl
+
+        z, x, y = (16, 13978, 28930)
+
+        self.generate_fixtures(
+            dsl.is_in("MX", z, x, y),
+            # https://www.openstreetmap.org/way/285597347
+            dsl.way(
+                285597347,
+                dsl.tile_diagonal(z, x, y),
+                {
+                    "lanes": "2",
+                    "name": "Guadalajara - Zapotlanejo",
+                    "oneway": "yes",
+                    "source": "openstreetmap.org",
+                    "toll": "yes",
+                    "ref": "MEX 15D",
+                    "highway": "motorway",
+                },
+            ),
+        )
+
+        self.assert_has_feature(
+            z, x, y, "roads", {
+                "id": 285597347,
+                "shield_text": "15D",
+                "network": "MX:MEX",
```

Comment:
Should this be `MX:MX` to follow 2-char convention elsewhere?

Reply:
Ah, ISO uses 3-char for subdivision codes for Mexican states so using `MEX` here would be more consisent with that. 👍